service.subtitles.subtitle/service.py

```diff
@@ -125,11 +125,13 @@ def takeTitleFromFocusedItem():
     labelTVShowTitle = xbmc.getInfoLabel("ListItem.TVShowTitle")
     labelSeason = xbmc.getInfoLabel("ListItem.Season")
     labelEpisode = xbmc.getInfoLabel("ListItem.Episode")
+    isItMovie = xbmc.getCondVisibility("Container.Content(movies)") or labelType == 'movie'
```

Comment:
might be that i overlooked that, i certainly dont spot every small detail when reviewing. Feel free to keep it this way, but it is conceptually wrong and will fail in other cases. Other add-ons doing it wrong is not really a good argument imo, someone has to start doing it right. ;)